app/Console/Commands/ForceDeleteTestRecords.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+
+class ForceDeleteTestRecords extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:forceDeleteTestRecords';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Force deletes signups and posts made in Runscope and Ghost Inspector tests';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Force delete all signups created by Runscope, identified by source.
+        $runscopeSignups = Signup::where('source', 'runscope')->orWhere('source', 'runscope-oauth')->get();
+        foreach ($runscopeSignups as $runscopeSignup) {
+            $runscopeSignup->forceDelete();
+        }
+
+        // Force delete all posts created by Runscope, identified by source.
+        $runscopePosts = Post::where('source', 'runscope')->orWhere('source', 'runscope-oauth')->get();
+        foreach ($runscopePosts as $runscopePost) {
+            $runscopePost->forceDelete();
+        }
```

Comment:
This chunk is a duplicate of the chunk above!

Reply:
@katiecrane this is for posts made by Runscope and the above is for signups made by Runscope!

Reply:
AH I see now! thanks!